specification/consumption/resource-manager/Microsoft.Consumption/stable/2019-10-01/examples/ReservationTransactionsListByBillingAccountId.json

```diff
@@ -1,67 +1,39 @@
 {
   "parameters": {
-    "api-version": "2019-10-01",
-    "billingAccountId": "123456",
-    "$filter": "properties/eventDate ge 2019-09-09 AND properties/eventDate le 2019-09-10"
+    "api-version": "2020-01-01",
```

Comment:
Why do you update the api-version in example file?

Reply:
That's a leftover from my previous attempt of adding a new api-Version. Updated it.